pkg/apis/crd/v1alpha1/types.go

```diff
@@ -709,3 +709,152 @@ type ExternalNodeList struct {
 
 	Items []ExternalNode `json:"items,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type SupportBundleCollectionList struct {
+	metav1.TypeMeta `json:",inline"`
+	// +optional
+	metav1.ListMeta `json:"metadata,omitempty"`
+
+	Items []SupportBundleCollection `json:"items"`
+}
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type SupportBundleCollection struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard metadata of the object.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Specification of the desired behavior of SupportBundleCollection.
+	Spec SupportBundleSpec `json:"spec"`
+	// Most recently observed status of the SupportBundleCollection.
+	Status SupportBundleStatus `json:"status"`
+}
+
+type SupportBundleSpec struct {
+	Nodes         *BundleNodes          `json:"nodes,omitempty"`
+	ExternalNodes []BundleExternalNodes `json:"externalNodes,omitempty"`
```

Comment:
Could we require a separate CR for each VPC/Namespace? I just want to simplify the API and implementation.

Reply:
Sure, I would update it.